regression/test.pl

```diff
@@ -71,7 +71,9 @@ ($$$$$)
 
   $options =~ s/$ign//g if(defined($ign));
 
-  my $output = $input;
+  my $descriptor = $test;
+  $descriptor =~ s/^.*\///;
```

Comment:
I agree with @smowton here that a comment `convert to path to basename` wouldn't hurt.